### ISSUE TITLE
Add an FAQ page about `MySQL lost connection error`.

### ISSUE THREAD

**Issue by toshihikoyanase** [optuna/optuna#622]
MySQL lost connection errors was fixed by the #504, but the instruction about `RDBStorage` argument (e.g., `{'pool_recycle': 30}`) is not documented. Please create an FAQ page about it. I think it is better to be explained somewhere in tutorials (e.g., [Saving/Resuming Study with RDB Backend](https://optuna.readthedocs.io/en/stable/tutorial/rdb.html)) as well.

**Comment by toshihikoyanase** [comment]
This issue will be resolved by #806, and we don't need to add the document mentioned in the above comment. But, it requires time to examine performance degradation due to the option introduced by #806.

For meanwhile, please provide the option to avoid connection errors here.

## Problem

When I use the MySQL server as the backend of the storage, I got `pymysql.err.OperationalError: (2013, 'Lost connection to MySQL server during query')`. 

**Code**

```python
import optuna
import time

url = 'mysql+pymysql://...'

def objective(trial):
    return trial.suggest_int('x', -10, 10)

study = optuna.create_study(storage=url)
study.optimize(objective, n_trials=1)
print('Best value: {} (params: {})\n'.format(study.best_value, study.best_params))

time.sleep(70)
study.optimize(objective, n_trials=1)  # The connection will be lost here.
```

## Workaround

Please set `engine_kwargs={'pool_pre_ping': True}` to `RDBStorage.__init__`. When we enable the `pool_pre_ping` option, SQL clients send `ping` to database servers to check connections are still viable before sending actual queries. If the connections are lost, the clients automatically reconnect to the servers. For further details of `pool_pre_ping` option, please consult [this SQLAlchemy document](https://docs.sqlalchemy.org/en/13/core/pooling.html#disconnect-handling-pessimistic).

**Code**

```python
import optuna
import time

url = 'mysql+pymysql://...'

def objective(trial):
    return trial.suggest_int('x', -10, 10)

storage = optuna.storages.RDBStorage(url, {'pool_pre_ping': True})  # Enable pre_ping here.
study = optuna.create_study(storage=storage)
study.optimize(objective, n_trials=1)
print('Best value: {} (params: {})\n'.format(study.best_value, study.best_params))

time.sleep(70)
study.optimize(objective, n_trials=1)  # This will successfully complete.
```

**Comment by toshihikoyanase** [comment]
#806 resolved this issue. Let me close it.
